cmake/modules/FindOracle.cmake

```diff
@@ -81,7 +81,8 @@ set(ORACLE_LIBRARIES ${ORACLE_LIBRARY})
 # Handle the QUIETLY and REQUIRED arguments and set ORACLE_FOUND to TRUE
 # if all listed variables are TRUE
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(ORACLE DEFAULT_MSG ORACLE_LIBRARY ORACLE_INCLUDE_DIR)
+find_package_handle_standard_args(Oracle DEFAULT_MSG ORACLE_LIBRARY ORACLE_INCLUDE_DIR)
+set(ORACLE_FOUND ${Oracle_FOUND})
```

Comment:
CMake variables are case-sensitive, so I assume that is why SOCI's CMake files have checks for both lower and uppercase names all over the place. I just wasn't sure if I would break something along the chain by changing this.

But if it's only used internally, I'd be down for switching to `Oracle_FOUND` only as well. This would also comply to the CMake standard: https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#standard-variable-names, which would mean that we should probably also use `Oracle_INCLUDE_DIR`, `Oracle_LIBRARY` etc. instead.
I'll look into it on the weekend.